testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/raw/RawHostedIT.java

```diff
@@ -36,6 +36,10 @@
 {
   public static final String HOSTED_REPO = "raw-test-hosted";
 
+  public static final String TEST_PATH = "alphabet.txt";
+
+  public static final String TEST_CONTENT = "alphabet.txt";
```

Comment:
This is making it consistent with the other raw ITs, I believe the reason is because `TEST_PATH` is where the raw resource will be deployed and `TEST_CONTENT` is where the test resource can be found relative to the test-data directory. At the moment these are the same, but might not be - see https://github.com/sonatype/nexus-oss/blob/NEXUS-6795-share-it-resources/testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/raw/RawGroupIT.java#L43 for example.